oremda/models/__init__.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseModel, Field
+from fastapi import WebSocket
+
+from oremda.typing import IdType
+
+
+class ObjectModel(BaseModel):
+    id: IdType = Field(...)
```

Comment:
Is `id: IdType = Field(...)` preferred over just `id: IdType`? It looks like they are the same: https://pydantic-docs.helpmanual.io/usage/models/#required-fields

Reply:
The code was just moved into a new file, may be @alesgenova can comment.

Reply:
Yeah they are the same, I like to put `Field(...)` explicitly so it can be easier to add configuration parameters if needed.

Either way works